### PR TITLE
PIM-4880 : use generated thumbnail cache for product PDF

### PR DESCRIPTION
**Pull request by jmleroux**
| Q                  | A
| ------------------ | ---
| Migration script?  |n
| Behats             |y
| Specs & Static     |y
| Changelog updated  |y

### CODE REVIEW

File: src/Pim/Bundle/PdfGeneratorBundle/Resources/views/Product/renderPdf.html.twig [hunk 1]
```diff
@@ -98,7 +98,7 @@
             {% block images %}
                 {% for attribute in imageAttributes if product.getValue(attribute.code, locale, scope).media is not null
                     and product.getValue(attribute.code, locale, scope).media.key is not null %}
-                    <img src="{{ url('pim_enrich_media_show', {'filename': product.getValue(attribute.code, locale, scope).media.key|url_encode, 'filter': 'thumbnail'}) }}" />
+                    <img src="media/cache/thumbnail/{{ product.getValue(attribute.code, locale, scope).media.key }}" />
```
Comment (sumbobyboys) [inline src/Pim/Bundle/PdfGeneratorBundle/Resources/views/Product/renderPdf.html.twig]: please use a route for that

**Comment by sumbobyboys** [comment]
GTM after comment fix

File: src/Pim/Bundle/PdfGeneratorBundle/Resources/views/Product/renderPdf.html.twig (hunk 1 above)
Comment (jjanvier) [inline src/Pim/Bundle/PdfGeneratorBundle/Resources/views/Product/renderPdf.html.twig]: he can't I think, that's the point of the fix..

**Comment by jjanvier** [comment]
gtm

File: src/Pim/Bundle/PdfGeneratorBundle/Resources/views/Product/renderPdf.html.twig (hunk 1 above)
Comment (jmleroux) [inline src/Pim/Bundle/PdfGeneratorBundle/Resources/views/Product/renderPdf.html.twig]: @sumbobyboys and @jjanvier : exact, i can't use a route.
I'd like to use a DI parameter or smth alike, but found nothing.

**Pull request merged**